laghos.cpp

```diff
@@ -886,6 +889,10 @@ int main(int argc, char *argv[])
         if (romOptions.sampX == 0 && !romOptions.mergeXV) romOptions.sampX = sFactorX * romOptions.dimX;
         if (romOptions.sampV == 0 && !romOptions.mergeXV) romOptions.sampV = sFactorV * romOptions.dimV;
         if (romOptions.sampE == 0) romOptions.sampE = sFactorE * romOptions.dimE;
+        /* Add this back. This makes sense, but it causes slight differences that make the regression tests fail.
```

Comment:
Will this be uncommented after the review?

Reply:
@kevinhkhuynh I propose that it be uncommented. It will change the regression testing baselines. I commented it out for the reviewers to be able to run the tests without failures. I have checked that this commented change results in different numbers of samples, causing small differences in the online solution.